ngraph/frontend/onnx_import/include/onnx_import/core/tensor.hpp

```diff
@@ -531,7 +531,7 @@ namespace ngraph
                 return static_cast<Type>(m_tensor_proto->data_type());
             }
 
-            const element::Type& get_ng_type() const
+            const element::Type get_ng_type() const
```

Comment:
Here I also recommend to return without the `const` specifier or change this method to return the `Type_t` enum instead.

Reply:
Fixed